README.md

```diff
@@ -44,3 +44,43 @@ or
 or
 
 `docker run -t -p 5000:5000 cache` (cache is what i named my docker image)
+
+Syntax note: in the `-p <host machine port>:<container port>`
+
+or
+
+`./run.sh docker`
+
+### Build Docker Image
+
+#### Setup Docker
+
+##### with Docker Beta
+
+Install the [Docker Beta](https://beta.docker.com/)
+
+##### with boot2docker (OSX)
+
+`brew install boot2docker` (It's vagrant for Docker containers, uses the hosts docker binary in a tiny VM)
+
+`boot2docker init`
+
+`boot2docker up` (Follow the trailing instructions for ENV vars OR cert install)
+
+#### Build the docker image
+`./build.sh docker`
+
+### Docker Compose
+
+There is also a Docker compose file (`docker-compose.yml`) that can be used to spin up
+the postgres database for getting the full hosted experience.
+
+`docker-compose build`
+
+`docker-compose up`
+
+#### Ports
+
+`http://localhost:5000` for the website
+
+`postgresql://localhost:5433/hucache` for the postgres instance (Connect with username `huboard` and password `huboard`)
```

Comment:
I haven't figured out how it works, but it's not in `hosts` as far as I can tell.